### PR TITLE
Deprecated existing phonebook API command, added new version

### PR DESCRIPTION
**Pull request by DrPyser**
https://wazo-dev.atlassian.net/browse/WAZO-3287

depends-on: https://github.com/wazo-platform/wazo-dird/pull/167

### CODE REVIEW

File: wazo_dird_client/client.py [hunk 1]
```diff
@@ -2,10 +2,29 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from wazo_lib_rest_client.client import BaseClient
+import wazo_dird_client.commands
 
 
 class DirdClient(BaseClient):
     namespace = 'wazo_dird_client.commands'
 
+    backends: wazo_dird_client.commands.backends.BackendsCommand
```
Comment (pc-m) [inline wazo_dird_client/client.py:11]: I don't understand why this is necessary. If you add the entry_points to the setup.py file all commands will be discovered by stevedore by the BaseClient class

**Review by pc-m — CHANGES_REQUESTED** [review]

File: wazo_dird_client/client.py (hunk 1 above)
Comment (DrPyser) [reply wazo_dird_client/client.py:11]: This is for typing and editor autocomplete. This has no runtime impact, but improves developer experience.

Comment (DrPyser) [reply wazo_dird_client/client.py:11]: The dynamic loading with stevedore hides the existence of those commands from any static analysis tooling(IDE, type checkers), which makes it harder to understand what you're dealing with when using this as library.

File: wazo_dird_client/client.py [hunk 2]
```diff
@@ -2,10 +2,29 @@
...
+    conference_source: wazo_dird_client.commands.conference_source.Command
+    config: wazo_dird_client.commands.config.ConfigCommand
+    csv_source: wazo_dird_client.commands.csv_source.Command
+    csv_ws_source: wazo_dird_client.commands.csv_ws_source.Command
+    directories: wazo_dird_client.commands.directories.DirectoriesCommand
+    displays: wazo_dird_client.commands.displays.DisplaysCommand
+    graphql: wazo_dird_client.commands.graphql.GraphQLCommand
+    ldap_source: wazo_dird_client.commands.ldap_source.Command
+    personal: wazo_dird_client.commands.personal.PersonalCommand
+    phonebook: wazo_dird_client.commands.phonebook.PhonebookCommand
```
Comment (pc-m) [inline wazo_dird_client/client.py:21]: phonebook_deprecated should be added here and in setup.py to be usable or be removed

**Review by pc-m — CHANGES_REQUESTED** [review]

**Review by pc-m — APPROVED** [review]

**Pull request merged**